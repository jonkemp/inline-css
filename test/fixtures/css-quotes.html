<html><link rel="stylesheet" href="css-quotes.css" /><body>
<p>woot</p></body></html>
