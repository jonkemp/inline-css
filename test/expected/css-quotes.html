<html><head></head><body>
<p style="background: url('/woot');">woot</p>
</body></html>
